radix-engine/README.md

```diff
@@ -0,0 +1,21 @@
+# Radix Engine
+
+Radix Engine is the underlying execution engine specialized to run DeFi-based Scrypto applications.
+
+## Architecture
+
+Radix Engine is organized into 7 layers.
+
+Each layer provides an API to the layer above.
+
+Execution layers may also optionally provide a Callback API which the layer above must implement.
+
+| Layer          | Responsibilities                                                                   | API                                                                                                                                                                                                                                                                          | Callback API                                             | Implementation(s)                                                                                          |
+|----------------|------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------------------------------|------------------------------------------------------------------------------------------------------------|
+| Blueprints     | Application Logic (e.g. Blueprints written in Scrypto)                             |                                                                                                                                                                                                                                                                              |                                                          | [Native Blueprints](src/blueprints)<br>[Scrypto Blueprints](../radix-engine-tests/tests/blueprints)        | 
```

Comment:
Is it worth prefacing the layer with either `Execution - ` or `Store - ` as per the diagram you showed in the meeting?

Reply:
I prefer simplicity in github md tables. If merged cells were easy I'd use that.